external/crypto-algorithms/src/sha256.c

```diff
@@ -32,8 +32,14 @@
 #define SIG0(x) (ROTRIGHT(x,7) ^ ROTRIGHT(x,18) ^ ((x) >> 3))
 #define SIG1(x) (ROTRIGHT(x,17) ^ ROTRIGHT(x,19) ^ ((x) >> 10))
 
+#if defined(__AVR__)
+#define PROGMEM_MARKER __flash
+#else
+#define PROGMEM_MARKER
+#endif
+
```

Comment:
We already have this in utils.h, don't we? Btw: What's the difference between `__flash` and the AVR-defined PROGMEM?

Reply:
I removed this again as there is plenty of space available in SRAM on our ATmega1284p and accessing SRAM is faster than accessing flash.

Reply:
For reference: Those were 512 Byte, so it is ok to use the SRAM here.